### PR TITLE
change 'avatar' to 'profile-photo' at design's request

### PR DESCRIPTION
**Pull request by awdyson**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
-@import "_typography";
 @import "_colors";
 @import "_outline";
+@import "_typography";
```
Comment (chinaowl) [inline src/index.scss:3]: Any particular reason for this switcheroo?

Comment (chinaowl) [reply src/index.scss:3]: Oh maybe because it's alphabetical now haha

Comment (awdyson) [reply src/index.scss:3]: Sorry, minor compulsion 😬

Comment (chinaowl) [reply src/index.scss:3]: I understand

**Pull request merged**